src/util/timeFormatter.js

```diff
@@ -0,0 +1,9 @@
+export function dateToString(date) {
+  if (date) {
+    return date.toLocaleTimeString(
+      'en-EN',
+      {weekday: 'long', year: 'numeric', month: 'long', day: 'numeric'},
+    );
+  }
+  return null;
+}
```

Comment:
Would be nice to fold this into the `i18next` fold, particularly so that if we ever do support other languages we won’t have to go rooting around too many places. Seems like i18n’s [`format`](https://www.i18next.com/translation-function/formatting) hook should work nicely?